docs/zh/latest/plugin-develop.md

```diff
@@ -25,7 +25,7 @@ title: 插件开发
 
 - [目录](#目录)
 - [检查外部依赖](#检查外部依赖)
-- [插件命名与配置](#插件命名与配置)
+- [插件命名，优先级和其他](#插件命名，优先级和其他)
```

Comment:
I'd like to know why we changed the title, BTW, the link doesn't work in GitHub Markdown.

Reply:
yes, In fact, I think it would be better to remove the `，` both in nav and docs ……